osrf/buoy_sim
# Spring Mean Piston Position

may now specify desired mean piston position in empy

to test, in a sourced terminal:

```
$ empy -D 'inc_wave_spectrum_type="None"' -o install/buoy_description/share/buoy_description/models/mbari_wec/model.sdf install/buoy_description/share/buoy_description/models/mbari_wec/model.sdf.em
$ ros2 launch buoy_gazebo mbari_wec.launch.py extra_gz_args:='-rs' regenerate_models:=false
```

and in another sourced terminal:
```
$ ros2 topic echo /spring_data
```

and check that range_finder settles close to the desired default value of 0.7m

can also define other values:
```
$ empy -D 'x_mean_pos=0.9' -D 'inc_wave_spectrum_type="None"' -o install/buoy_description/share/buoy_description/models/mbari_wec/model.sdf install/buoy_description/share/buoy_description/models/mbari_wec/model.sdf.em
```

should settle close to 0.9m

Also added to batch parameters and model regen in tests

TODO:
- figure out what I did wrong in free-body forces (unexpected fudge factor required)
